spec/support/databse_cleaner.rb

```diff
@@ -0,0 +1,5 @@
+# frozen_string_literal: true
+
+RSpec.configure do |rspec|
+  rspec.before(:suite) { DatabaseCleaner.clean_with :truncation }
```

Comment:
Why do we need this?

Reply:
Cypress is cleaning up only before each test, meaning that rspec running after cypress starts with a few records in the database from the last cypress test. This was causing quite a few unexpected failures